int8 user_number
string body_part
---
float64[] transformation
float64[] rotation
